HIP/hip-904-frictionless-airdrop.md

```diff
@@ -464,34 +772,91 @@ Scenario 2: If Dave submits a TokenClaim for token T5 and a Token claim of seria
 
 `automatic_token_associations` containing
 
-- Daves automatic association with token T5
-- Dave automatic association with token T6
+- Dave's automatic association with token T5
+- Dave's automatic association with token T6
 
 `pending_airdrops` will be empty
 
-Note: NFT serial 6 of T6 is left in pending airdrop, but all of the pending fungible amount of T5 was transferred 
+Note: NFT serial 6 of T6 is left in pending airdrop, but all of the pending fungible amount of T5
+was transferred.
 
-Scenario 3: Eric submits a TokenCancelAirdrop for token T7 and a TokenCancelAirdrop of serial 7 of T8
+Scenario 3: Alice submits a TokenCancelAirdrop for token T7 and a TokenCancelAirdrop of serial 7 of T8
 
 `tokenTransferLists` will be empty
 
 `automatic_token_associations` will be empty
 
 `pending_airdrops` will be empty, details are in transaction body
 
-Note: NFT serial 6 of T6 is left in pending airdrop, but all of the pending fungible amount of T5 was transferred 
+Note: NFT serial 8 of T8 is left in pending airdrop, and none of the pending fungible amount of T7
+was transferred.
 
 #### Reject
 
+> REVIEW NOTE
```

Comment:
Yes, we want the ability to reject serials ids specifically due the the potential to have a large balance of them.
For FTs it's easy but for NFTs the network processing each serial could be involving.